tests/integration/mixer/telemetry/logs/accesslog_test.go

```diff
@@ -94,9 +93,6 @@ func testsetup(ctx resource.Context) error {
 		return err
 	}
 	galInst = &g
-	if _, err = mixer.New(ctx, mixer.Config{Galley: g}); err != nil {
```

Comment:
Is this factored in with the operator already ?
Or was this just validating ?

Reply:
this is not doing anything really except port forwarding to mixer then doing nothing. It's not actually installing mixer. I forget if I made this change to fix anything or if I was cleaning this up, I have been working on this off and on for a couple months